### PR TITLE
[v52] STAC-18277: Customisable monitors titles

### PR DESCRIPTION
**Pull request by mBut**

### CODE REVIEW

File: develop/developer-guides/custom-functions/monitor-functions.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ The script should return a result of type `MonitorHealthState` with the followin
 * `state` - A `HealthStateValue`. This will be the new health state of the monitor (`CLEAR`, `DEVIATING`, `CRITICAL`, `DISABLED` or `UNKNOWN`).
 * `topologyIdentifier` - The identifier of a component or relation that the monitor health state will bind to.
 * `displayTimeSeries` - Description of a time series that will be shown as a chart in the StackState UI.
+* `title` (optional) - A title to display for the monitor-produced health states.
```
Comment (gggina) [inline develop/developer-guides/custom-functions/monitor-functions.md:86]: `A title to display for health states produced by the monitor.`

Where will this title be displayed?

Comment (mBut) [reply develop/developer-guides/custom-functions/monitor-functions.md:86]: Screenshot 2023-01-05 at 10 58 28
It's also important to note that if no title is provided than the monitor name is used instead. How would you rephrase it to make it more clear?
